framework/src/mysql/mysql.js

```diff
@@ -321,6 +322,31 @@ const getTableInstance = async (...tableParams) => {
 			query.orWhereIn(property, values);
 		}
 
+		if (params.leftOuterJoin) {
+			const { targetTable, joinColumnLeft, joinColumnRight } = params.leftOuterJoin;
+			query.leftOuterJoin(targetTable, joinColumnLeft, joinColumnRight);
+		}
+
+		if (params.rightOuterJoin) {
+			const { targetTable, joinColumnLeft, joinColumnRight } = params.rightOuterJoin;
+			query.rightOuterJoin(targetTable, joinColumnLeft, joinColumnRight);
+		}
+
+		if (params.innerJoin) {
+			const { targetTable, joinColumnLeft, joinColumnRight } = params.innerJoin;
+			query.innerJoin(targetTable, joinColumnLeft, joinColumnRight);
+		}
+
+		if (params.crossJoin) {
+			const { targetTable, joinColumnLeft, joinColumnRight } = params.crossJoin;
+
+			if (joinColumnLeft && joinColumnRight) {
+				query.crossJoin(targetTable, joinColumnLeft, joinColumnRight);
+			} else {
+				query.crossJoin(targetTable);
+			}
+		}
```

Comment:
As cross join is only supported by mysql, should we add it?
https://knexjs.org/guide/query-builder.html#crossjoin

Reply:
Should we remove cross join in that case?